sdk/src/wallet/utxo.ts

```diff
@@ -279,33 +310,51 @@ export async function estimateTxFee(
     // TODO: allow submitting the UTXOs, fee estimate and confirmed transactions
     // to avoid fetching them again.
     const esploraClient = new EsploraClient(addressInfo.network);
+    const ordinalsClient = new OrdinalsClient(addressInfo.network);
 
     let confirmedUtxos: UTXO[] = [];
+    let outputsFromAddress: OutputJson[] = [];
```

Comment:
Would leave a comment here that this contains UTXOs which *do not* contain inscriptions. I assume that's right from reading the code below anyway

Reply:
comment added